tests/test_integration.py

```diff
@@ -632,6 +632,18 @@ def test_self_is_not_installed(projdir, flag, usedevelop):
     assert 'test==0.0.0' not in result.stdout
     assert 'test @ file://' not in result.stdout
 
+@pytest.mark.parametrize("externals", [None, "allowlist_externals", "whitelist_externals"])
+def test_externals(projdir, externals):
+    with modify_config(projdir / 'tox.ini') as config:
+        if externals:
+            config['testenv'][externals] = "foo"
+    result = tox("-e", NATIVE_TOXENV, "--current-env", quiet=False)
+    assert result.returncode == 0
+    assert any(
+        "test command found but not installed in testenv" not in x
+        for x in (result.stdout, result.stderr))
```

Comment:
I find this `any` a bit confusing. What do we assert here? If this text should never be present, this should be `all`, no?